src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+- Custom Policy: Custom policies allow you to implement additional business rules, using templates that you define, to enforce policies that are not already addressed using the other built-in policy types. For more information, see [Creating Custom Policies](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-1FF7A1E5-8456-4EF4-A532-9CF31BE88EAA.html).
+
+Some policies mentioned here are not available in Tanzu Mission Control Standard edition. For more information, see the [VMware Tanzu Mission Control Feature Comparison Chart](https://tanzu.vmware.com/content/tanzu-mission-control/tmc-comparison-chart).
+
+#### Policy Inheritance
+
+In the Tanzu Mission Control resource hierarchy, you can specify policies at the following three levels:
+
+- Organization
+- Object groups (cluster groups and workspaces)
+- Kubernetes objects (clusters and namespaces)
+
+#### Global Visibility example
+
+With TMC you can group clusters based on different types of locations and manage fleets of clusters instead of managing them individually. When we are dealing with large scale edge deployments this becomes very critical as customers might have two or three different types of locations. Being able to group them with cluster groups, looking at individual fleet health, performing collective operations is key for edge deployments. Figure X shows an overview of the cluster group.
+
+![](./img/tkg-edge/ce29c1e20bf0ecc1ffe2dfdc658cb585.png)
+
+Figure: Managing the edge cluster fleet using TMC
+
+#### Policy Management example
+
+Policies allow you to provide a set of rules that govern your organization and all the objects it contains. Policies can be applied to fleets of clusters by defining once to ensure that the kubernetes clusters and applications are secure, compliant while enabling self service which makes it very appealing for edge scale.  Tanzu mission control provides following policy types that can be applied to fleets of clusters:
+
+1. Access policy
+2. Image registry policy
+3. Network policy
+4. Quota policy
+5. Pod Security Policy.
+
+Figure X and Figure X below showcase how customers can create policies and use the policy insights page to review if any of the policies are violated across all the edge deployments.
+
+![](./img/tkg-edge/a93e92ad27bee927a258c5447a3ed78d.png)
+
+Figure: Using TMC to manage edge cluster policies
+
+![](./img/tkg-edge/559bfe3fa044f89dab66d033ee2168cf.png)
+
+Figure: Policy reporting across all edge locations
+
+#### Usage consideration
+
+As of time of writing [Tanzu Mission Control][701] requires,  at least, ~1.5GB of memory and ~1 vCPU per managed cluster to operate.
+
```

Comment:
The following table describes the resources used by Tanzu Mission Control components.